code/game/turfs/turf.dm

```diff
@@ -62,6 +62,22 @@ GLOBAL_LIST_EMPTY(created_baseturf_lists)
 	///Icon-smoothing variable to map a diagonal wall corner with a fixed underlay.
 	var/list/fixed_underlay = null
 
+	///Is this tile high traction / non slip
+	var/traction = FALSE
+
+	///Ref to texture mask overlay
+	var/texture_mask_overlay
+
+	///What turf texture we use
+	var/list/turf_texture = list()
```

Comment:
lazylist this. Or, honestly, I'm of the opinion that this doesn't need to be a list at all right now. Nothing uses more than one afaik currently and lists are slow for init.

Reply:
Multiple areas use it as a list
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/40559528/672a5fd4-bd25-4c21-9086-ed26c4fc0552)

Reply:
I'll start it as null, and only areas that use it will instantiate it as a list